src/overlays/actors/ovl_En_Part/z_en_part.h

```diff
@@ -6,13 +6,15 @@
 
 struct EnPart;
 
+typedef void (*EnPartActionFunc)(struct EnPart*, GlobalContext*);
+
 typedef struct EnPart {
     /* 0x000 */ Actor actor;
-    /* 0x14C */ u8 unk_14C;
-    /* 0x14E */ s16 unk_14E;
-    /* 0x150 */ s32 unk_150;
-    /* 0x154 */ f32 unk_154;
-    /* 0x158 */ f32 unk_158;
+    /* 0x14C */ u8 action;
+    /* 0x14E */ s16 effectDelay;
```

Comment:
I could be wrong but I think this is just the timer for the whole actor
```suggestion
    /* 0x14E */ s16 timer;
```